local-cluster/Main.hs

```diff
@@ -18,13 +24,16 @@ import Test.Plutip.LocalCluster (
 
 main :: IO ()
 main = do
+  args <- Options.execParser (info (pClusterConfig <**> helper) mempty)
   (st, _) <- startCluster def $ do
-    w <- addSomeWallet [toAda 10000]
+    let nWall = numWallets args
+        wPath = dirWallets args
+    ws <- replicateM (max 1 nWall) $ addSomeWalletDir [toAda 10000] wPath
```

Comment:
> max 1 nWall

I don't think this single wallet is required anymore.
Before this module was more like example of how to make executable which starts local cluster and creates single wallet.
But now we give ready to go executable to user, so user can decide how many wallets needed via args, IMO

Reply:
Also, I think we should let user to decide how many Ada UTxOs create as well as amount of Ada in them.
Now each wallet will have single UTxO with 10000 Ada.